frontend/src/app/main/ui/workspace/libraries.cljs

```diff
@@ -104,9 +107,45 @@
       (tr "workspace.libraries.typography" typography-count)])])
 
 
+(mf/defc sample-library-entry
+  [{:keys [library project-id team-id importing] :as props}]
+  (let [id         (:id library)
+        importing? (deref importing)
```

Comment:
All new react components should follow the new style (name ends in `*` and it should have `::mf/props` to `:obj` and referenced/used with `[:>`)

Reply:
I'm also thinking that passing the whole state `importing` is the way to implement this. I mean, in general is not a good practice, but there are always exceptions.

Reply:
On the end, I think it is ok for this case, so no change is required for my previous comment